src/test/groovy/org/prebid/server/functional/tests/SmokeSpec.groovy

```diff
@@ -197,4 +201,78 @@ class SmokeSpec extends BaseSpec {
         then: "Response should contain bidders params"
         assert response.isEmpty()
     }
+
+    def "PBS should return by default blank when we use usersync.iframe without format-override"() {
```

Comment:
It doesn't comfortably make 1 or 2 tests when we use different containers parameter and it will big where close statement, as for me better when more than 1-2 test but more understandable.

Reply:
I'd agree with Alex here. I'd rather have 2 parametrized tests for no-override and override scenarios. This looks more logical, but will require some modifications to existing enums.